parser.go

```diff
@@ -1057,15 +1056,14 @@ func (p *parser) parseSchemaObject(pkgPath, pkgName, typeName string) (*SchemaOb
 		itemTypeName := typeName[5:]
 		schema, ok := p.KnownIDSchema[genSchemeaObjectID(pkgName, itemTypeName)]
 		if ok {
-			schemaObject.Items = &SchemaObject{Ref: addSchemaRefLinkPrefix(schema.ID)}
+			schemaObject.AdditionalProperties = &SchemaObject{Ref: addSchemaRefLinkPrefix(schema.ID)}
```

Comment:
I don't remember all the context, but when I was digging into this, according to printf, we never went down this path (`ok` was never true) for things like `FlagConfigurationRep` in Gonfalon.

But it's likely I was doing something wrong. Were you able to run this against gonfalon? Did the ref work?

Reply:
Haven't tried this against Gonfalon yet, but this is a known bug.
https://app.clubhouse.io/launchdarkly/story/102258/goas-ref-not-fully-working

Reply:
Oh, "nice"!